### DISCUSSION TITLE
xwayland fixed an issue of repeated key presses.

### DISCUSSION THREAD

**Discussion by amano-kenji** [fcitx/fcitx5#638] [Q&A] [answered]
https://gitlab.freedesktop.org/xorg/xserver/-/issues/1395 fixes the issue in xwayland-22.1.4 that resembles https://github.com/fcitx/fcitx5/issues/496

Perhaps, https://github.com/fcitx/fcitx5/issues/496 was an xwayland issue?

**Comment by wengxt** [comment, marked as answer]
I believe it's a separate issue. But I think I also hit this issue with xwayland application sometimes. It's nice to see that it's fixed.
